@xen-orchestra/fs/src/local.js

```diff
@@ -5,9 +5,9 @@ import { fromEvent, retry } from 'promise-toolbox'
 import RemoteHandlerAbstract from './abstract'
 
 export default class LocalHandler extends RemoteHandlerAbstract {
-  constructor(remote, { triesOnEAGAIN = 10 } = {}) {
+  constructor(remote, { triesOnEagain = 10 } = {}) {
     super(remote)
-    this._triesOnEAGAIN = triesOnEAGAIN
+    this._triesOnEagain = triesOnEagain
```

Comment:
IMHO, `retries` would be better because the first try is not *on EAGAIN*.

Reply:
`retriesOnEagain` is clearer.

Reply:
And you can pass full options to give configuration more flexibility: https://github.com/vatesfr/xen-orchestra/blob/853905e52f0e3d0fc9cfb5d03342d6bc25783d25/packages/xen-api/src/index.js#L89-L94